src/solutions/ode_solutions.jl

```diff
@@ -224,33 +255,87 @@ function (sol::AbstractODESolution)(t::Number, ::Type{deriv}, idxs::AbstractVect
         error("Incorrect specification of `idxs`")
     end
     error_if_observed_derivative(sol, idxs, deriv)
-    interp_sol = augment(sol.interp([t], nothing, deriv, sol.prob.p, continuity), sol)
-    first(interp_sol[idxs])
+    ps = parameter_values(sol)
+    # NOTE: This is basically SII.parameter_values_at_time but that isn't public API
+    # and once we move interpolation to SII, there's no reason for it to be
+    if is_parameter_timeseries(sol) == Timeseries()
```

Comment:
I had expected a function with the name `is_parameter_timeseries` to return a boolean, answering the question of the function name.

Reply:
Yeah, it's actually a trait function necessary for dispatching which is why it returns a singleton object

Reply:
In that case, `parameter_timeseries_trait` would perhaps be better? 

A function starting with `is` but not returning a bool is very confusing

Reply:
Yeah that work work. I initially chose `is_parameter_timeseries` because we already have `is_timeseries` which (unfortunately) is also a trait function (for the `IsTimeseriesTrait`, which `is_parameter_timeseries` also leveranges)